docs/plugins.rst

```diff
@@ -177,6 +177,27 @@ then be able to log in with their Google account when they click the log in page
 and select the option to log in with Google.
 
 
+Curation
+--------
+
+This plugin adds curation functionality to Girder, similar to that in MIDAS.
```

Comment:
Since people (like me) don't immediately know what "curation" means in this case and may not know about MIDAS, I'd suggest replacing "similar to that in MIDAS" with a sentence or two about what it means to curate data. Something like "This enables administrators to give certain users write access to folders while maintaining control of how its contents change through an extra approval process." I'm sure this could be explained better.